src/Turing.jl

```diff
@@ -28,6 +29,8 @@ Enable progress logging in Turing if `progress` is `true`, and disable it otherw
 function setprogress!(progress::Bool)
     @info "[Turing]: progress logging is $(progress ? "enabled" : "disabled") globally"
     PROGRESS[] = progress
+    AbstractMCMC.setprogress!(progress; silent=silent)
```

Comment:
I think we want
```suggestion
    AbstractMCMC.setprogress!(progress; silent=true)
```
regardless of whether the keyword argument `silent` of `Turing.setprogress!` is `true` or `false`: If it is `true`, then AbstractMCMC should be silent as well; and if it is `false`, then the info from Turing is already sufficient.